packages/dds/tree/src/test/tablePerformanceTestUtilities.ts

```diff
@@ -103,70 +105,32 @@ export function createTableTree(tableSize: number, cellValue: string): TableTree
 	);
 
 	treeView.initialize(Table.empty());
-	const undoStack: Revertible[] = [];
-	const redoStack: Revertible[] = [];
-
-	function onDispose(disposed: Revertible): void {
-		const redoIndex = redoStack.indexOf(disposed);
-		if (redoIndex !== -1) {
-			redoStack.splice(redoIndex, 1);
-		} else {
-			const undoIndex = undoStack.indexOf(disposed);
-			if (undoIndex !== -1) {
-				undoStack.splice(undoIndex, 1);
-			}
-		}
-	}
-
-	const unsubscribeFromCommitAppliedEvent = treeView.events.on(
-		"commitApplied",
-		(commit, getRevertible) => {
-			if (getRevertible !== undefined) {
-				const revertible = getRevertible(onDispose);
-				if (commit.kind === CommitKind.Undo) {
-					redoStack.push(revertible);
-				} else {
-					undoStack.push(revertible);
-				}
-			}
-		},
-	);
-	const unsubscribe = (): void => {
-		unsubscribeFromCommitAppliedEvent();
-		for (const revertible of undoStack) {
-			revertible.dispose();
-		}
-		for (const revertible of redoStack) {
-			revertible.dispose();
-		}
-	};
 
 	const table = treeView.root;
 	for (let i = 0; i < tableSize; i++) {
-		const column = new Column({ id: `column-${i}` });
+		const column = new Column({});
 		table.insertColumn({ index: i, column });
 	}
 	for (let i = 0; i < tableSize; i++) {
-		const row = new Row({ id: `row-${i}`, cells: {} });
+		const row = new Row({ cells: {} });
 		table.insertRow({ index: i, row });
 	}
-	for (let i = 0; i < tableSize; i++) {
-		for (let j = 0; j < tableSize; j++) {
+	for (const row of table.rows) {
+		for (const column of table.columns) {
 			table.setCell({
 				key: {
-					column: `column-${i}`,
-					row: `row-${j}`,
+					column,
+					row,
 				},
-				cell: { cellValue },
+				cell: cellValue,
 			});
 		}
 	}
-
 	return {
 		table,
 		undoStack,
```

Comment:
If this function is no longer setting up undo/redo behavior, do we need to initialize and return these empty stack arrays? Seems like we could initialize them in the place where we set up undo/redo.